### PR TITLE
#16756 update rename test

### PR DESCRIPTION
**Pull request by Crazykev**
update `integration-cli/docker_cli_rename_test.go`,  use gocheck asserts instead of fatal   #16756

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks for helping out @Crazykev !

**Comment by HuKeping** [comment]
building error @Crazykev

**Comment by thaJeztah** [comment]
```
# github.com/docker/docker/integration-cli
./docker_cli_rename_test.go:63: cannot use err (type error) as type string in argument to check.Commentf
./docker_cli_rename_test.go:67: cannot use err (type error) as type string in argument to check.Commentf
./docker_cli_rename_test.go:71: cannot use err (type error) as type string in argument to check.Commentf
FAIL	github.com/docker/docker/integration-cli [build failed]
```

**Comment by Crazykev** [comment]
Does this mean that my test is not successful ?  @thaJeztah

**Comment by thaJeztah** [comment]
@Crazykev hm, experimental passed, but the others didn't, but you can see the full logs of failed builds using the "details" links above. In this case its an integration-test that's failing; I haven't looked good into that, but it could be related

If you don't think it's related, you can ask in the #docker-dev IRC channel if a maintainer can manually restart the build for your PR

**Comment by HuKeping** [comment]
You can use `git commit --amend` and then `git push -f` to rebuild it also.

**Comment by thaJeztah** [comment]
@HuKeping true, but if just one of the builds fails, it's faster to only restart that build

Ignore the Windows builds for now, looks like they're all failing currently (something wrong with the CI)

**Comment by Crazykev** [comment]
@thaJeztah @HuKeping THX for your advice, I'll try and learn how to use IRC.

File: integration-cli/docker_cli_rename_test.go [hunk 1]
```diff
@@ -19,12 +20,8 @@ func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
...
 
 	name, err = inspectField(cleanedContainerID, "Name")
-	if err != nil {
-		c.Fatal(err)
-	}
-	if name != "/"+newName {
-		c.Fatal("Failed to rename container ", name)
-	}
+	c.Assert(err, check.IsNil, check.Commentf(name))
+	c.Assert(name != "/"+newName, check.Equals, false, check.Commentf("Failed to rename container %s", name))
```
Comment (cpuguy83) [inline integration-cli/docker_cli_rename_test.go]: Woah this is funky to read.
How about `c.Assert(name, check.Equals, "/"+newName)`

File: integration-cli/docker_cli_rename_test.go [hunk 2]
```diff
@@ -53,36 +46,31 @@ func (s *DockerSuite) TestRenameCheckNames(c *check.C) {
...
 
 	name, err := inspectField(newName, "Name")
-	if err != nil {
-		c.Fatal(err)
-	}
-	if name != "/"+newName {
-		c.Fatal("Failed to rename container ")
-	}
+	c.Assert(err, check.IsNil, check.Commentf(name))
+	c.Assert(name != "/"+newName, check.Equals, false, check.Commentf("Failed to rename container %s", name))
```
Comment (cpuguy83) [inline integration-cli/docker_cli_rename_test.go]: Same as above.

File: integration-cli/docker_cli_rename_test.go [hunk 3]
```diff
@@ -53,36 +46,31 @@ func (s *DockerSuite) TestRenameCheckNames(c *check.C) {
...
-		c.Fatalf("Renaming container to empty name should have failed: %s\n%v", out, err)
-	}
+	out, _, err = dockerCmdWithError("rename", "", "newname")
+	c.Assert(err, check.NotNil, check.Commentf("Renaming container with empty name should have failed: %s", out))
+	c.Assert(out, checker.Contains, "may be empty", check.Commentf("%v", err))
 
-	if out, _, err := dockerCmdWithError("ps", "-a"); err != nil || !strings.Contains(out, "myname") {
-		c.Fatalf("Output of docker ps should have included 'myname': %s\n%v", out, err)
-	}
+	out, _, err = dockerCmdWithError("ps", "-a")
```
Comment (cpuguy83) [inline integration-cli/docker_cli_rename_test.go]: This sould just be `dockerCmd`

**Comment by Crazykev** [comment]
Thanks for your advice @cpuguy83

**Comment by Crazykev** [comment]
updated @cpuguy83

**Comment by vdemeester** [comment]
@Crazykev just a nit, but it would be perfect if we used `checker` for any checker (`checker.IsNil`, etc…). Other than that, looks good, thanks ! :wink:

**Comment by Crazykev** [comment]
@vdemeester  yep, it would reduce some confusion.

**Comment by cpuguy83** [comment]
LGTM

**Comment by estesp** [comment]
LGTM

**Comment by vdemeester** [comment]
:metal:

**Comment by Crazykev** [comment]
**First commit in github.** :metal:

**Comment by xlgao-zju** [comment]
:metal:

**Comment by thaJeztah** [comment]
Great work @Crazykev :tada:

https://twitter.com/thajeztah/status/652521443020959744

**Pull request merged**